lblod/app-lokaal-mandatenbeheer
# fix missing min max number of members

## Description

fixes the missing min/max number of members.
## How to test

You can verify that the only organs missing min max values are the bcsds of faciliteitengemeenten (which will be removed later) using this query:

```
  PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
  PREFIX org: <http://www.w3.org/ns/org#>
  PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
  PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
  PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>

  SELECT distinct ?eenheidLabel ?role ?label ?mandaat WHERE {
    GRAPH ?g {
      ?mandaat a mandaat:Mandaat.
      ?orgInT org:hasPost ?mandaat.
      ?orgInT lmb:heeftBestuursperiode <http://data.lblod.info/id/concept/Bestuursperiode/96efb929-5d83-48fa-bfbb-b98dfb1180c7>.
      FILTER NOT EXISTS {
        ?mandaat lmb:minAantalHouders ?something.
      }

    }

    VALUES ?role {
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000011>
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000014>
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000015>
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000017>
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000019>
      <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e00001b>
    }

    ?mandaat org:role ?role.
    ?role skos:prefLabel ?label.

    ?g ext:ownedBy ?eenheid.
    ?eenheid skos:prefLabel ?eenheidLabel.

  } 
```
